test/Microsoft.ApplicationInsights.AspNetCore.Tests/Extensions/ApplicationInsightsExtensionsTests.cs

```diff
@@ -371,6 +371,17 @@ public static void AddsQuickPulseProcessorToTheConfigurationInFullFrameworkWithS
                 var updatedCount =  GetTelemetryProcessorsCountInConfiguration<QuickPulseTelemetryProcessor>(telemetryConfiguration);
                 Assert.Equal(updatedCount, exisitingProcessorCount + 1);
             }
+
+            [Fact]
+            public static void ProcessorsAreNotAddedToTheConfigurationWithExistingNonServerChannel()
+            {
+                var exisitingProcessorCount = GetTelemetryProcessorsCountInConfiguration<QuickPulseTelemetryProcessor>(TelemetryConfiguration.Active);
```

Comment:
Please try to avoid declaring everything with "var", especially for primitives like int and for the return values of methods, as this greatly decreases code readability.